TUM Campus App/NewsManager.swift

```diff
@@ -60,9 +60,7 @@ class NewsManager: Manager {
             return a.date.compare(b.date as Date) == ComparisonResult.orderedDescending
         }
         if single {
-            if let firsStory = items.first {
```

Comment:
This could crash if the news array is empty... My suggestion:

```Swift
if let firstStory = items.filter({ $0.date > now }).last ?? items.first {
    handler([firstStory])
}
```

Or make the result of getNextUpcomingNews an optional. But I would prefer to never have anything with indexes in the code. Makes it safer